gp-unique-id/gpuid-global-sequential-id.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Gravity Wiz // GP Unique ID // Share a global sequential unique ID across all forms
+ * https://gravitywiz.com/documentation/gravity-forms-unique-id/
+ *
+ * Sets a globally sequential unique ID across all forms on the website. 
+ * To use this snippet, add Unique ID fields on all the forms you'd like to generate Unique IDs, 
```

Comment:
Add an additional line break to separate the long description from the instructions.